source/backend/api/Services/CompensationRequisitionService.cs

```diff
@@ -146,6 +146,14 @@ public IEnumerable<PimsCompensationRequisition> GetFileCompensationRequisitions(
             return compReqs;
         }
 
+        public IEnumerable<PimsCompReqFinancial> GetCompensationRequisitionFinancials(long id)
+        {
+            _logger.LogInformation("Getting compensations financials for id: {acquisitionFileId}", id);
+            _user.ThrowIfNotAuthorized(Permissions.CompensationRequisitionView, Permissions.CompensationRequisitionView);
```

Comment:
not sure why we would have the same permission twice here - please remove duplicate